style.css

```diff
@@ -718,22 +710,86 @@
     overflow-x: hidden;
   }
 
+  .head {
+    display: block;
+  }
+
   .nav-top {
-    display: flex;
+    display: block;
   }
 
   .tit {
     position: relative;
-    right: 90px;
+    right: 120px;
+    top: 14px;
   }
 
   .nav-menu {
```

Comment:
- [ ] [OPTIONAL] You can give the menu a ``` position ``` absolute and ```width``` and ```height``` of 100% so the menu is on top even when you scroll the page